_site/assets/css/custom.css

```diff
@@ -0,0 +1,86 @@
+#va-team-gallery .gallery-divider {
```

Comment:
Do not use a custom.css file 

Do create proper scss files that correspond to either Page specific styles or component styles